software/ensemblvep/main.nf

```diff
@@ -0,0 +1,62 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options = initOptions(params.options)
+params.use_cache = false
+params.vep_tag = ""
+
+process ENSEMBLVEP {
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::ensembl-vep=104.3" : null)
+    if (params.use_cache) {
+        if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+            container "https://depot.galaxyproject.org/singularity/ensembl-vep:104.3--pl5262h4a94de4_0"
+        } else {
+            container "quay.io/biocontainers/ensembl-vep:104.3--pl5262h4a94de4_0"
+        }
+    } else {
+        container "nfcore/vep:${params.vep_tag}"
+    }
+
+    input:
+    tuple val(meta), path(vcf)
+    val (vep_genome)
+    val (vep_species)
+    val (vep_cache_version)
+    path (vep_cache)
```

Comment:
```suggestion
    tuple val(meta), path(vcf)
    val   genome
    val   species
    val   cache_version
    path  cache
```

Reply:
We can use short names here because the `vep` is inferred from the module itself?